src/pykoala/instruments/koala_ifu.py

```diff
@@ -184,15 +186,20 @@ def read_rss(file_path,
                log=log,
                )
 
-def koala_rss(path_to_file):
+def koala_rss(path_to_file, **kwargs):    #!!! Added **kwargs for verbose, plot
     """
     A wrapper function that converts a file (not an RSS object) to a koala RSS object
     The paramaters used to build the RSS object e.g. bad spaxels, header etc all come from the original (non PyKoala) .fits file
     """
+    vprint('\n> Converting RSS file',path_to_file,'to a koala RSS object...', **kwargs) #!!!
     header = fits.getheader(path_to_file, 0) + fits.getheader(path_to_file, 2)
     koala_header = py_koala_header(header)
     # WCS
-    koala_wcs = WCS(header)
+    koala_wcs = WCS(header, relax=False)   #!!!  Ángel Added relax = False to avoid annoying warning:
```

Comment:
Just a suggestion. There is no need to write in the code the changes you've made if you create a commit to reflect them. This keeps the code cleaner while still having track of the changes

Reply:
I think `relax=True` tells the code to be "permissive", and try fix any obsolete keywords or problems it finds. On the other hand, `relax=False` means "be strict and don't tolerate any error or inconsistency, no matter how minimal". Therefore, the code should not only give a warning, but raise an exception (I guess).

To suppress warnings within a given scope, try:
```
with warnings.catch_warnings():
    warnings.simplefilter('ignore', AstropyWarning)
    koala_wcs = WCS(header)
```
<https://docs.astropy.org/en/stable/warnings.html>

Reply:
I'm marking the changes in the code mainly FOR ME. I want to be sure I'm not breaking anything or can easily put it back to what it was. I'll clean everything once it is good.

Again, I spent some time trying to get rid of the annoying error message...

Added at the beginning of the code:

```
import warnings
from astropy.utils.exceptions import AstropyWarning
```

and then call it with:

```
with warnings.catch_warnings():
        warnings.simplefilter('ignore', AstropyWarning)
        koala_wcs = WCS(header) 
```